app/views/NotificationPreferencesView/index.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import {
+	View, ScrollView, SafeAreaView, Switch, Text
+} from 'react-native';
+import PropTypes from 'prop-types';
+import RNPickerSelect from 'react-native-picker-select';
+
+import { SWITCH_TRACK_COLOR } from '../../constants/colors';
+import StatusBar from '../../containers/StatusBar';
+import ListItem from '../../containers/ListItem';
+import Separator from '../../containers/Separator';
+import I18n from '../../i18n';
+import scrollPersistTaps from '../../utils/scrollPersistTaps';
+import styles from './styles';
+import sharedStyles from '../Styles';
+import database from '../../lib/realm';
+import RocketChat from '../../lib/rocketchat';
+import log from '../../utils/log';
+
+const SectionTitle = React.memo(({ title }) => <Text style={styles.sectionTitle}>{title}</Text>);
+
+const SectionSeparator = React.memo(() => <View style={styles.sectionSeparatorBorder} />);
+
+const Info = React.memo(({ info }) => <Text style={styles.infoText}>{info}</Text>);
+
+SectionTitle.propTypes = {
+	title: PropTypes.string
+};
+
+Info.propTypes = {
+	info: PropTypes.string
+};
+
+const OPTIONS = {
+	desktopNotifications: [{
+		label: I18n.t('Default'), value: 'default'
+	}, {
+		label: I18n.t('All Messages'), value: 'all'
```

Comment:
Typo here and on others
<img src="https://user-images.githubusercontent.com/804994/63286166-c0109400-c28d-11e9-8591-8067d58f9073.png" width="350" />